mne/evoked.py

```diff
@@ -358,7 +358,7 @@ def plot_field(self, surf_maps, time=None, time_label='t = %0.0f ms',
         return plot_evoked_field(self, surf_maps, time=time,
                                  time_label=time_label, n_jobs=n_jobs)
 
-    def plot_white(self, noise_cov, show=True):
+    def plot_white(self, noise_cov, rank=None, show=True):
```

Comment:
should be after `show` for backward compat

Reply:
Do we really support API positional arguments?
I feel something like show should be the last one. And people should anyways not use positional arguments. The order of the purely positional arguments should be of course preserved.

Reply:
> Do we really support API positional arguments?

Yes, at least for the first few inputs (say maybe 10?). In this case there were only two so I think it's clear.

> I feel something like show should be the last one. 

We'd need to do this in a separate PR

>  And people should anyways not use positional arguments.

I think it's reasonable to use positional arguments and expect API stability there. Imagine someone has code:
```
show = True  # can be toggled to false in iteration

...
evoked.plot_white(cov, show)
```
This seems fine to me, and this change will break it.

Reply:
I don't share this view as I always recommend to pass by keyword, but if you insist it's ok with me.

Reply:
Yes please